docs/tables/aws_elastcache_cluster.md

```diff
@@ -0,0 +1,88 @@
+# Table: aws_elasticache_cluster
+
+A cluster is a collection of one or more cache nodes, all of which run an instance of the Redis cache engine software.
+
+## Examples
+
+### List of unencrypted elasticache clusters
```

Comment:
```suggestion
### List unencrypted clusters
```

Reply:
I expanded the description a bit since below we have an example for checking encryption in transit, so good to specify this is for encryption at rest